app/views/ucas_contacts/index.html.erb

```diff
@@ -0,0 +1,197 @@
+        <dd class="govuk-summary-list__value">
+          <p class="govuk-body" data-qa="provider__web_link_contact">
+            <% if @provider.web_link_contact.present? %>
+              <%= @provider.web_link_contact[:name] %>
+              <%= @provider.web_link_contact[:email] %>
+              <%= @provider.web_link_contact[:telephone] %>
+            <% else %>
+              Information unknown
+            <% end %>
+          </p>
+          <p class="govuk-body-s">
+            This person will receive notices about UCAS systems and planned downtime
+          </p>
+        </dd>
+        <dd class="govuk-summary-list__actions"></dd>
+      </div>
+
+      <div class="govuk-summary-list__row">
+        <dt class="govuk-summary-list__key">
+          Finance contact
+        </dt>
+        <dd class="govuk-summary-list__value">
+          <p class="govuk-body" data-qa="provider__finance_contact">
+            <% if @provider.finance_contact.present? %>
+              <%= @provider.finance_contact[:name] %>
+              <%= @provider.finance_contact[:email] %>
+              <%= @provider.finance_contact[:telephone] %>
+            <% else %>
+              Information unknown
+            <% end %>
+          </p>
+          <p class="govuk-body-s">
+            This person will receive invoices from UCAS for payment of fees
+          </p>
+        </dd>
+        <dd class="govuk-summary-list__actions"></dd>
+      </div>
+
+      <div class="govuk-summary-list__row">
+        <dt class="govuk-summary-list__key">
+          Fraud contact
+        </dt>
+        <dd class="govuk-summary-list__value">
+          <p class="govuk-body" data-qa="provider__fraud_contact">
+            <% if @provider.fraud_contact.present? %>
+              <%= @provider.fraud_contact[:name] %>
+              <%= @provider.fraud_contact[:email] %>
+              <%= @provider.fraud_contact[:telephone] %>
+            <% else %>
+              Information unknown
+            <% end %>
+          </p>
+          <p class="govuk-body-s">
+            This person will receive alerts about fradulent activity (for example when a plagiarised personal statement is detected)
+          </p>
+        </dd>
+        <dd class="govuk-summary-list__actions"></dd>
+      </div>
+
+      <div class="govuk-summary-list__row">
+        <dt class="govuk-summary-list__key">
+          UCAS Administrator
+        </dt>
+        <dd class="govuk-summary-list__value">
+          <p class="govuk-body" data-qa="provider__admin_contact">
+            <% if @provider.admin_contact.present? %>
+              <%= @provider.admin_contact[:name] %>
+              <%= @provider.admin_contact[:email] %>
+              <%= @provider.admin_contact[:telephone] %>
+            <% else %>
+              Information unknown
+            <% end %>
+          </p>
+          <p class="govuk-body-s">
+            This person is responsible for setting up and maintaining UCAS user accounts.
```

Comment:
We could DRY these rows:

```ruby
[
  ["UCAS Teacher Training (UTT) correspondent", :utt_contact, "This person will receive the UCAS monthly bulletin, which includes information about deadlines for application decisions"],
  ["web-link correspondent", :web_link_contact, "This person will receive notices about UCAS systems and planned downtime"],
  ...
].each do ...
```

What do you think? It would make the view shorter,  and we could store the array in the decorator.

Though I think we will end up editing this page very seldomly, so I don't think it's of high importance to do this.

Reply:
Great shout! I'll this in phase two 💪